doc/man3/CRYPTO_THREAD_run_once.pod

```diff
@@ -25,6 +32,28 @@ CRYPTO_atomic_add, CRYPTO_atomic_or, CRYPTO_atomic_load - OpenSSL thread support
                       CRYPTO_RWLOCK *lock);
  int CRYPTO_atomic_load(uint64_t *val, uint64_t *ret, CRYPTO_RWLOCK *lock);
 
+ void CRYPTO_mem_barrier(void);
```

Comment:
There does not seem to be a description of this function

Reply:
Thanks! Added.